app/views/search/_collection_details.html.erb

```diff
@@ -81,7 +81,14 @@
         </div>
 
         <div class="row">
-          <div class="col-sm-12 long-paragraph collapsed" data-bind="text: $data.description"></div>
+          <div class="col-sm-12 long-paragraph collapsed">
```

Comment:
I think you can and should implement this in CSS rather than adding code to the backend and frontend:

    .long-paragraph {
        white-space: pre-line;
    }